src/dex-ui-components/themes/layerStyles.ts

```diff
@@ -47,4 +47,12 @@ export const LayerStyles = {
   wizard__footer: {
     width: "100%",
   },
+  dao__dashboard__card: {
```

Comment:
We're using [BEM](https://getbem.com/naming/) for style naming (block__element--modifier). So, this variant name should be `dao-dashboard__card` (block = `dao-dashboard`, element = `card`)

CC: @roshan-bisht-lab49 Tagging you as this info may be useful for future tasks.

Reply:
Changed the name as pr your suggestion.